cmd/process-agent/collector.go

```diff
@@ -118,6 +121,16 @@ func (l *Collector) run(exit chan bool) {
 	}
 	log.Infof("Starting process-agent for host=%s, endpoints=%s, orchestrator endpoints=%s, enabled checks=%v", l.cfg.HostName, eps, orchestratorEps, l.cfg.EnabledChecks)
 
+	if err := l.forwarder.Start(); err != nil {
+		log.Errorf("Error starting forwarder: %s", err)
+		return
+	}
+
+	if err := l.podForwarder.Start(); err != nil {
```

Comment:
Should we only do this if the `orchestrator` check is enabled?

Reply:
I guess we could, we would have to surround this and the stop below with an if check.  Is it worth it to avoid having idle goroutines sitting around?